README.md

```diff
@@ -67,9 +69,11 @@ yarn db:local:migrate
 ### Dependencies <a name="dependencies"></a>
 You will need to have the following installed:
 
-[Node JS](https://nodejs.org/en/download/releases/) ( LTS Erbium v14.x )
+[Node JS](https://nodejs.org/en/download/releases/) ( LTS Erbium v18.x )
```

Comment:
[Node 18 LTS is called 'Hydrogen'](https://nodejs.org/en/download/releases)

Reply:
Thanks 👍🏾 have changed that now